### PR TITLE
UI/UX changes made

### PR DESCRIPTION
**Pull request by Aiu26**
#25 Added and improved the design of the app

### CODE REVIEW

**Comment by harshit1142** [comment]
@Aiu26 add some videos/ screenshots .

File: .all-contributorsrc [hunk 1]
```diff
@@ -1,75 +1,85 @@
 {
```
Comment (harshit1142) [inline .all-contributorsrc]: No need to modify this file.

File: contributors/contributorsList.js [hunk 2]
```diff
@@ -1,22 +1,27 @@
 contributors = [
```
Comment (harshit1142) [inline contributors/contributorsList.js:1]: this also , no need to modify

**Comment by Aiu26** [comment]
Uploading 04.10.2024_00.23.02_REC.mp4…

Video of the UI

**Comment by Aiu26** [comment]
https://github.com/user-attachments/assets/e7c14035-5301-46c0-b35a-ba596ff0a81d

Uploaded Video of UI

**Comment by harshit1142** [comment]
@Aiu26 Great !!

**Review by harshit1142 — APPROVED** [review]

**Comment by Aiu26** [comment]
please add hacktoberfest as a label as well

**Pull request merged**